tests/wpt/web-platform-tests/innerText/setter.html

```diff
@@ -9,6 +9,7 @@
 // tests a not-rendered case.
 
 function setupTest(context, plain) {
+  var container = document.getElementById("container");
```

Comment:
nit: You can move this outside of the function if you want.

Reply:
`container` is only used inside that function and having the var definition there kind of matches with the `detachedContainer` var inside `setupTestDetached`, so I think it looks better there :)